contents/docs/product-analytics/identify.mdx

```diff
@@ -295,14 +295,16 @@ PostHogSDK.shared.identify("distinct_id_two",
 </MultiLanguage>
 
 
-### How to handle duplicates of the same user
+### How to merge users
 
 It may happen that, due to implementation issues, the same user in your product has multiple users in PostHog associated with them. In these cases, you can use `$merge_dangerously` to merge multiple PostHog users into a single user.
    
-> **Warning:** Merging users with `merge_dangerously` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together.
+> **⚠️ Warning:** Merging users with `merge_dangerously` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together.
 > 
 > Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems.
 
+> **⚠️ Warning:** Before using `merge_dangerously`, first check if [alias](/docs/product-analytics/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) would better suit your needs.
```

Comment:
Instead of multiple callouts, can we just have one? We could also just have the second point in the paragraph above this warning. 
```suggestion
> **⚠️ Warnings:** 
> 1. Merging users with `$merge_dangerously` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together. Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems.
> 2. Before using `$merge_dangerously`, first check if [alias](/docs/product-analytics/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) would better suit your needs.
```